docs/development/UX/strategy.md

```diff
@@ -0,0 +1,38 @@
+# UX strategy
+
+## Comfort of use
+
+The device should be pleasant to use. Users would have to hold it together with phone and flip this stack over, probably several times. We should not add unnecessary difficulties on top of that. Things like metadata management should happen automatically and without users' help.
```

Comment:
It is helpful to have picture here. I show some possibilities on photo, but still not sure what you meant by "flip" position. I would suggest to work around [book, slide-0, slide-1] positions since they are nearly interchangeable with each other (they can be used with the same screen text direction), and also slide-1 seems the most natural.  

Should text direction be vertical or horizontal? It is better decide while we are testing a physical device. 

![Kampela + phone positions](https://user-images.githubusercontent.com/69721131/194913904-f66504fe-1e86-4037-9076-773b9e0b05d9.jpeg)

Reply:
Form factor
I think that the implementation of the device in the form of a phone case, for example, in the form of a book, will make it possible to justify the organization of the production of the device on a pre-order basis using 3D printing technology in manufacturing.

Reply:
Re-designing the casing means that we should re-do security analysis every time. We should separate the base device from extra convenience casing to be able to do this. Anyway, let's not put this into strategy, as this is not and could not be decided now, before mockup tests.